virtool/samples/db.py

```diff
@@ -82,18 +82,26 @@ async def attach_artifacts_and_reads(pg: AsyncEngine, document: dict) -> dict:
     :param document: A document that represents a sample
     :return: Updated document with associated sample artifacts
     """
+    sample = document["_id"]
```

Comment:
This is clearer in code even though the corresponding column is called `sample`.

```suggestion
    sample_id = document["_id"]
```